.travis/mvnsettings.xml

```diff
@@ -0,0 +1,23 @@
+<settings>
+	<servers>
+		<server>
+			<id>ossrh</id>
+			<username>${env.OSSRH_JIRA_USERNAME}</username>
+			<password>${env.OSSRH_JIRA_PASSWORD}</password>
+		</server>
+	</servers>
+
+	<profiles>
+		<profile>
+			<id>ossrh</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<properties>
+				<gpg.executable>gpg</gpg.executable>
+                <gpg.keyname>${env.GPG_KEYNAME}</gpg.keyname>
```

Comment:
space used on this line although tabs are used on all others

Reply:
ok

Reply:
Updated